client/me/security-account-recovery/edit-email.jsx

```diff
@@ -1,6 +1,7 @@
 /**
  * External dependencies
  */
+var PropTypes = require('prop-types');
```

Comment:
So far so good!
I noticed that some imports use `var`, a little weird but that's hardly worth blocking this :)

Reply:
I figured we could fix this with the imports codemod after the fact :)